packages/remark-shaku-code-annotate/README.md

```diff
@@ -98,6 +108,10 @@ pre.shiki .line.dim {
   opacity: 0.3;
 }
 
+pre.shiki .line.focus {
+  opacity: 0.3;
+}
```

Comment:
This is not quite right, you are adding `.focus` which  does the job of `.dim`

Reply:
> This is not quite right, you are adding `.focus` which does the job of `.dim`

The reason I use the .focus class is as follows:

Users can adjust the opacity of de-emphasized lines marked with @focus using .focus, and adjust the opacity of de-emphasized lines marked with @dim using .dim, without interfering with each other.

Alternatively, we could provide users with just one class name to adjust the opacity, either .dim or .focus.

Which option do you think we should choose, or do you have any other better solutions? Thank you!

Reply:
I'd prefer there be only one `.dim` class.
`.focus` for most of the case does nothing. unless people really want to add something fancy. 

So we can just remove these CSS rules, but add `.focus` class in script.

Reply:
> I'd prefer there be only one `.dim` class for this purpose `.focus` for most of the case does nothing. unless people really want to add something fancy.
> 
> So we can just remove these CSS rules, but add `.focus` class in script, like a placeholder.

Okay, I will fix it later